README.md

```diff
@@ -28,6 +28,26 @@ Tracer.initGlobalTracer(LightStep.tracer({
 
 * **Node**: Node versions >= 0.12 are supported.
 
+## LightStep Initialization Options
+
+### Browser-specific
+
+#### `xhr_url_inclusion_patterns RegExp[]`
```

Comment:
I agreed that there is a need to set better operation names for specific URLs / URL patterns.  I'm not sure I want to include it as part of this option, but this may be something to revisit.